### PR TITLE
Docs for CCI and CCI divergence

### PR DESCRIPTION
**Pull request by quantum8**
Docs for CCI, CCI divergence and MFI divergence.

### CODE REVIEW

File: Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-004-CCI-Divergence.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        },        
+		{
+            "style": "Text",
+            "text": "Disagreement between the indicator and price is called divergence, and it can have significant implications for trade management. The amount of agreement/disagreement is relative, so there can be several different patterns that develop in the relationship between price and the indicator."
+        },
+        {
+            "style": "Link",
+            "text": "Understanding momentum - (investopedia)->www.investopedia.com/trading/trading-divergence-and-understanding-momentum/#toc-momentum-divergence"
+        },
+		{
```
Comment (gitforkgit) [inline Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-004-CCI-Divergence.json:25]: Did you manually indent the opening brackets to the rigth?
And do you want to delete the last file?

Comment (quantum8) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-004-CCI-Divergence.json:25]: For line 14? can't remember but I may have.

Which file are you referring to?

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-004-CCI-Divergence.json:25]: Line 17 in the file i highlighted above.

Comment (gitforkgit) [reply Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-004-CCI-Divergence.json:25]: Can you regularly app.update?

**Comment by quantum8** [comment]
Looks like base version did a major update between when i commented on this pull request and now. I usually keep it up to date, and am now 6 commits ahead of main dev branch.

**Review by gitforkgit — APPROVED** [review]

**Pull request merged**